CRM/Contact/BAO/Query.php

```diff
@@ -4908,6 +4908,23 @@ public function searchQuery(
     return $dao;
   }
 
+  /**
+   * Create and query the db for a contact search.
+   *
+   * @return CRM_Core_DAO
+   */
+  public function alphabetQuery() {
+    $query = $this->getSearchSQL(NULL, NULL, NULL, FALSE, FALSE, TRUE,
+      FALSE, NULL, NULL, NULL, FALSE);
```

Comment:
@mattwire can you pull out these later parameters that match the defaults - I'm trying to eliminate some of them from the signature so being able to see which calling fns are passing what is better

Reply:
@eileenmcnaughton Done